ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+<!--
+
+Thank you for taking time to open an issue.
+Please take time to read the instructions, even if you've read it before, as things may have changed in the recent past.
+
+There are three main reasons you may want to open an issue here:
+
+1. a bug report
+2. a feature request
+3. operational issues
+
+#################################################################################
+IN ALL OF THESE CASES, PLEASE SEARCH FOR RELATED ISSUES BEFORE OPENING A NEW ONE.
+#################################################################################
+
+For bug reports, please retain the "BUG REPORT" portion of this template and discard the rest.
+For feature requests, please retain the "FEATURE REQUEST" portion, and discard the rest.
+
+#################################################################################
+DO NOT OPEN A NEW ISSUE ABOUT OPERATIONAL ISSUES
+#################################################################################
+
+This repository is not meant for operaional issues. Please check the status on https://www.traviscistatus.com/, and
+if you believe it is not reflected, please email us at support@travis-ci.com instead.
+
+-->
+
+
+
+
+<!-- ##### BEGIN BUG REPORT ##### -->
+
+# Pre-issue checklist
+- [ ] I have searched GitHub issues for potentially related issues
+- [ ] I have read and searched https://docs.travis-ci.com/, in particular, https://docs.travis-ci.com/user/common-build-problems/
+
+<!-- If you have searched for bugs in this repository and still have not found a related issue, include: -->
+# Brief description of the bug
+
+# Steps to reproduce the bug
+
+## Build log URL to show the bug
+<!-- If you do not wish to disclose the build log URL, that is fine. Please email support@travis-ci.com **INSTEAD OF** opening a new issue here. -->
+
+# Expected behavior
+
+# Actual behavior
+
+<!-- If this is a community-supported language, be sure to mention its maintainers. -->
+
+<!-- ##### END BUG REPORT ##### -->
+
+
+
+
+
+<!-- ##### BEGIN FEATURE REQUEST ##### -->
+<!-- If you have searched in this repository and still have not found an issue that matches your feature request, include: -->
+
+# Brief description of the feature
+
+<!-- ##### END FEATURE REQUEST ##### -->
+
```

Comment:
Who labels issues? I'll add a blurb here at the bottom explaining that we'll triage and tag their issue accordingly and follow-up ?  

Maybe in the readme it'd be nice to explain how to tag and triage? That would likely help with contributions on this front :)